cypress/pages/BaseBuilder.js

```diff
@@ -1,6 +1,6 @@
 export class BaseBuilder {
   get saveButton() {
-    return cy.contains("Save to File System");
+    return cy.contains("Save to File System", { timeout: 60 * 1000 });
```

Comment:
Is the page very slow to load?

Reply:
It did on my machine, I can try and remove it and see what happens :)

Reply:
https://github.com/prismicio/slice-machine/actions/runs/4292541292